adoc/chapters/programming_interface.adoc

```diff
@@ -6044,7 +6044,7 @@ For example, the following could be used:
 
 However, optionally the [code]#buffer::set_final_data()# can be set to a
 [code]#std::weak_ptr# to enable copying data back, to another host memory
-address that is going to be valid after buffer construction.
+address that is going to be valid after buffer destruction.
```

Comment:
If we want to commit this as an interim improvement, let's change this to something like:

> ... that will be valid when the buffer is destroyed.

Saying "after buffer destruction" is wrong because the pointer must be valid before the buffer destructor is invoked.

Reply:
The latest commit makes this change, so I think the conversation is resolved (but @gmlueck should confirm)

Reply:
Thanks, @mkinsner